packages/react-ui-core/src/index.js

```diff
@@ -91,11 +91,6 @@ export {
   PhotoCarousel,
 } from './Carousel'
 
-export {
```

Comment:
You sure this isn't used by anyone else?

Reply:
Yes, I checked rent-js and rentals-js and neither were importing these components.